.github/workflows/check_values_schema.yaml

```diff
@@ -0,0 +1,41 @@
+name: 'check if values schema file has been updated'
+on: pull_request
+
+jobs:
+  check:
+    name: 'check files in PR'
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'checkout'
+        uses: actions/checkout@v2
+      - id: files
+        uses: jitterbit/get-changed-files@v1
+      - name: 'show changed files in PR'
+        run: echo "${{ steps.files.outputs.all }}"
+      - name: 'check if values.schema.json was updated'
+        run: |
+          VALUES_FILE_CHANGED="false"
+          SCHEMA_FILE_CHANGED="false"
+
+          if find -name values.schema.json | grep -q "values.schema.json" ; then
```

Comment:
```suggestion
          if test -n "$(ls values.schema.json)"; then
```

Reply:
`find` needs a 'starting-point`, also without `-maxdepth` can lead to things being slow if dir is deep. Also, I personally try to avoid pipes, since an error on LHS can lead to unexpected behaviour

Reply:
find has always worked for me with and without a location to search - if you don't specify it it defaults to the current dir (the root of the repo checkout).

the difficulty with adding maxdepth is that this action should work on any app repo and the charts can sometimes be in lower dirs depending on how it has been written. i _could_ make it faster by setting `-type f` so it ignores all other types.

because of the fact that the check runs from the root of the checkout and because i don't know how many dirs deep the file is located, i can't use `ls` because the schema file won't be in the dir the action runs from.

Reply:
from the manpage:

> If no starting-point is specified, `.' is assumed.

admittedly this is on Arch so i don't know when this was added/changed, but it's always worked for me regardless of OS for the last few years :shrug:

Reply:
besides, this check will need to be reworked/become more complex because of the possibility of there being multiple matching files in the repo

Reply:
Not knowing where the file might be (i was assuming it always been known or top level), then I guess find should be OK.... However, if this is a git repo we are searching through (I am guessing it is), you could also use ` git ls-tree -r HEAD` (Can replace `HEAD` with whatever branch). That would list all files managed by git